extensions/wikia/PortableInfobox/querypage/AllinfoboxesQueryPage.php

```diff
@@ -106,9 +107,20 @@ public function reallyDoQuery( $limit = false, $offset = false ) {
 			} );
 
 		return array_filter( $result, function ( $tmpl ) {
-			$data = PortableInfoboxDataService::newFromPageID( $tmpl[ 'pageid' ] )->getData();
+			$title = Title::newFromID( $tmpl[ 'pageid' ] );
 
-			return !empty( $data );
+			if ( $title && $title->exists() &&
```

Comment:
In order to make this logic flow easier to read, please consider making this conditional a method that returns boolean ( i.e. isValidInfoboxPageTitle($title) )